projects/m2/014-sublists/js/index.js

```diff
@@ -0,0 +1,27 @@
+function isSublist(largerList, smallerList){
+    if( largerList.length >  smallerList.length){
+        if( Array.isArray( smallerList ) && smallerList.length === 0 ){
+            return true;
+        }
+
+        const firstElementindex = largerList.indexOf(smallerList[0]);
+
+        if( firstElementindex !== -1 ){
+            for (let i = 0; i < smallerList.length; i++) {

```

Comment:
You could try to fix this using the **forEach** method ;-)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
Hi, I fixed the exercise.